.circleci/config.yml

```diff
@@ -2,12 +2,12 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.3.8-node-browsers
+      - image: circleci/ruby:2.6.3-node-browsers
         environment:
           RAILS_ENV: test
           AWS_DEFAULT_REGION: us-west-2
           EB_APP_NAME: ingest
-      - image: mdillon/postgis:9.5
+      - image: mdillon/postgis:11
```

Comment:
I think that the generated schema.sql may have used features present in Postgres 11 because I had 11 installed on my dev box. Not sure what it would take to walk it back to 9.5, but it shouldn't be too hard to upgrade the RDS databases, I think.
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_UpgradeDBInstance.PostgreSQL.html

Reply:
yeah, should be mostly point and click as long as the current app runs on 11. Should be easy to upgrade ingest-dev to confirm